devdox/app/routes/git_tokens.py

```
@@ -7,21 +7,22 @@
-from fastapi import APIRouter, Body, Depends, Query, Request, status
+from fastapi import APIRouter, Body, Depends, Request, status
 from starlette.responses import JSONResponse
 
 import app.exceptions.exception_constants
 from app.config import GitHosting
 from app.exceptions.exception_constants import SERVICE_UNAVAILABLE
 from models.git_label import GitLabel
 from models.user import User
-from app.schemas.basic import PaginationParams
-from app.schemas.git_label import GitLabelBase, GitLabelCreate
+from app.schemas.basic import PaginationParams, RequiredPaginationParams
```

Ruff (0.11.9)

`app.schemas.basic.RequiredPaginationParams` imported but unused

Remove unused import: `app.schemas.basic.RequiredPaginationParams`

(F401)